src/ServantElm.hs

```diff
@@ -98,7 +129,7 @@ endpointInfoToElmQuery requestInfo =
     typeSignature =
       mkTypeSignature requestInfo
 
-    args = hsep ["urlBase", "toMsg"]
+    args = hsep ["urlBase", "toMsg", "bodyValue"]
```

Comment:
The `"bodyValue"` constant is used in two places and it's necessary for it to be the same in both places - you should extract it as a constant.

Reply:
done, did the same to the other argument values